src/audio/lib/aisetnextbuf.c

```diff
@@ -1,4 +1,5 @@
-#include "global.h"
+#include "ultra64.h"
+#include "z64audio.h"
```

Comment:
What's z64audio.h needed for here?

Reply:
nothing, idk how this happened. thanks for catching it